ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/application/Hook.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.application
+
+/**
+ * Represents a hook that can be registered in [ApplicationPluginBuilder].
+ */
+@PluginsDslMarker
+public interface Hook<HookHandler> {
+    /**
+     * Specifies how to install a hook in the [application].
+     */
+    public fun install(application: Application, config: HookHandler)
```

Comment:
`config` -> `handler`?

Reply:
Thank you, fixed

Reply:
Still not fixed, here and everywhere.
Another name could be `configure`.
imo, `config` = `configuration` = state / data class
`configure` = action
`handler` = callable object, also action